### ISSUE TITLE
Masthead overflow – overflow button not appearing/disappearing on tab

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#4183]
## Detailed description

> Describe in detail the issue you're having.

- **Included the `sprint must have` label as this overflow feature is committed for the Nov release** 
When using the keyboard to tab through the masthead items, the left overflow button does not appear after the items shift to the right and the first item is partially hidden. 

The left overflow button should appear, and the right overflow button should disappear.

![left arrow does not appear on tabOct-08-2020 11-56-56](https://user-images.githubusercontent.com/45692622/95486515-18f32e00-0961-11eb-8276-8328c25676ad.gif)

Link to [functional specs](https://ibm.ent.box.com/file/728119889086?s=tm0ouknu0fw84w2ibfctuig5bub2f9it)
Link to [visual specs](https://ibm.ent.box.com/file/716827406287?s=00ieobl9wsxjsxb3rhs7y6o7mccurx5r)see page 32–35 for visual reference

> Is this a feature request (new component, new icon), a bug, or a general
> issue?

bug

> Is this issue related to a specific component?

masthead

> What did you expect to happen? What happened instead? What would you like to
> see changed?

> What browser are you working in?

Chrome
Version 85.0.4183.121 (Official Build) (64-bit)

> What version of the IBM.com Library are you using?

Canary

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

October release

## Steps to reproduce the issue

1. use keyboard to tab through items in masthead
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes

**Comment by jeffchew** [comment]
This might be a similar issue that was brought up by Ram and @Praveen-sr , that screen readers actually do not need the overflow buttons. Related: #4182

**Comment by oliviaflory** [comment]
The arrow on the right side should disappear once the user has tabbed to the last item within the navigation, there are no other links to the right of "explore more" in this example, so the arrow is misleading/useless. I don't think showing and labeling the overflow button when there is no additional content makes sense for the user. cc @Praveen-sr @jeffchew

**Comment by RobertaJHahn** [comment]
Ram will share his accessibility recommendation in the Oct 22 Prod QA meeting.

**Comment by RobertaJHahn** [comment]
Engineers will implement as documented (Olivia to be sure that documentation reflects her design.).  Ram will review after code is available.

**Comment by RobertaJHahn** [comment]
@Praveen-sr Can you retest this to be sure the change works properly.  Thanks.

**Comment by Praveen-sr** [comment]
@oliviaflory @RobertaJHahn We re tested this on [React](https://ibmdotcom-react-canary.mybluemix.net/?path=/story/components-masthead--with-l-1) 
Checked on:
iPad 8th, iPad mini and iPad Pro [IOS 13, 14]
Samsung Tab S7 and Tab S6 [Android 9, 10]
Browser re sizing- Firefox, chrome and Safari

1. User using keyboard to tab through the masthead menu items, the left overflow button does appear after the items shift to the right 
2. User using keyboard to tab through the masthead menu items, right overflow button disappears once user reaches last item 
3. User using keyboard to Shift + tab through the masthead menu items, left overflow button disappears once user reaches first item.

 Working as expected.

**Comment by RobertaJHahn** [comment]
@Praveen-sr Thanks for your help.  Closing.

cc:  @oliviaflory @ariellalgilmore
